ui/components/multichain/multichain-token-list-item/multichain-token-list-item.stories.js

```diff
@@ -29,7 +29,7 @@ export default {
   },
   args: {
     secondary: '$9.80 USD',
-    primary: '88.00687889',
```

Comment:
primary will always have max 4 decimal values. Hence, we don't need to define this case because in `asset-list.js` which is the parent component for the native token list. We have defined ethNumberOfDecimals to be 4 here 

`  const {
    currency: primaryCurrency,
    numberOfDecimals: primaryNumberOfDecimals,
  } = useUserPreferencedCurrency(PRIMARY, { ethNumberOfDecimals: 4 });`